### ISSUE TITLE
Zabbix server and web cannot start correctly

### ISSUE THREAD

**Issue by quangnhut123** [zabbix/zabbix-community-docker#56]
Hi,
I just using this Docker file and see if we run the configuration not correctly in DB password when we passing it to paramater.

Issued command : 

```
docker run -d --name zabbix -p 80:80 -p 10051:10051 \
-v /etc/localtime:/etc/localtime:ro \
--link zabbix-db:zabbix.db --env="ZS_DBHost=zabbix.db" \
--env="ZS_DBUser=zabbix" \
--env="ZS_DBPassword=P@ssword123!" \
monitoringartist/zabbix-3.0-xxl:latest
```

The password will be fill to ''P@ssword123!'' in #1 and 'P@ssword123!' for #2. Its incorrect. Please fix it to 'P@ssword123!' for #1 and P@ssword123! for #2
Check file #1: /usr/local/src/zabbix/frontends/php/conf/zabbix.conf.php
Check file #2: /usr/local/etc/zabbix_server.conf (DBPassword)

Thanks

**Comment by jangaraj** [comment]
Fix available in `monitoringartist/zabbix-3.0-xxl:dev` image for now:

```
[root@dockerhost ~]# docker run -d --name zabbix --env='ZS_DBPassword=P@ssword123!' monitoringartist/zabbix-3.0-xxl:dev
e96d93837cbdf749512cff633e029019d492fb3a6ac61d316accfaf5e5c10855
[root@dockerhost ~]# docker exec -ti zabbix grep @ /usr/local/src/zabbix/frontends/php/conf/zabbix.conf.php
$DB['PASSWORD'] = 'ZS_P@ssword123!';
[root@dockerhost ~]# docker exec -ti zabbix grep @ /usr/local/etc/zabbix_server.conf
DBPassword=P@ssword123!
```

**Comment by jangaraj** [comment]
Done. Fixed images are available.
